admin/registration_providers/forms.py

```diff
@@ -20,9 +20,18 @@ class RegistrationProviderForm(forms.ModelForm):
 
     class Meta:
         model = RegistrationProvider
-        exclude = ['primary_identifier_name', 'primary_collection', 'type', 'allow_commenting', 'advisory_board',
-                   'example', 'domain', 'domain_redirect_enabled', 'reviews_comments_anonymous',
-                   'reviews_comments_private', 'reviews_workflow', 'collected_type_choices', 'status_choices']
```

Comment:
Drive-by: I think we don't want `reviews_comments_private` for registries. I also think @jwalz is going to want to do something different from `reviews_workflow`, so I'm pinging him to see if we should keep excluding this field.

Reply:
Planning on still using reviews_workflow, for consistency, just overriding the inherited field to limit the choices